bin/check-elb-health-sdk.rb

```diff
@@ -47,13 +45,18 @@ class ELBHealth < Sensu::Plugin::Check::CLI
          short: '-r AWS_REGION',
          long: '--aws-region REGION',
          description: 'AWS Region (defaults to us-east-1).',
-         default: 'us-east-1'
+         default: ''
```

Comment:
@sstarcher I've set default back to 'us-east-1', and instead use keyword 'all' for specify go through all EC2 regions.